openslides_votecollector/views.py

```diff
@@ -187,6 +187,17 @@ def no_error_context(self):
         """
         return {}
 
+    @staticmethod
```

Comment:
It doesn't depend on any local (self) attributes. Therefore I consider it a static method. Alternatively, it could be a view package method: view.get_polles